slides/pages/login.ftd

```diff
@@ -69,13 +69,12 @@ align-content: center
 icon: $assets.files.slides.assets.github-mark-white.svg
 link: $login-page.github-url
 
-
 -- ftd.row:
 spacing.fixed.px: 6
 align-content: center
 padding.px: 10
 
--- ds.copy-small: Don’t have an account?
+-- ds.copy-regular: Don’t have an account?
```

Comment:
@MeenuKumari28 all strings must be in `en.ftd` and `hi.ftd`, do not leave any user visible string in other files.